## 常见问题

### 超时错误的解决访问，scrapy 的解决方式
```
2017-11-15 11:07:30 [scrapy.core.scraper] ERROR: Error downloading <GET https://wh.lianjia.com/zufang/jiangan/>
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/twisted/internet/defer.py", line 1384, in _inlineCallbacks
    result = result.throwExceptionIntoGenerator(g)
  File "/usr/local/lib/python3.6/dist-packages/twisted/python/failure.py", line 408, in throwExceptionIntoGenerator
    return g.throw(self.type, self.value, self.tb)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/core/downloader/middleware.py", line 43, in process_request
    defer.returnValue((yield download_func(request=request,spider=spider)))
  File "/usr/local/lib/python3.6/dist-packages/twisted/internet/defer.py", line 653, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/usr/local/lib/python3.6/dist-packages/scrapy/core/downloader/handlers/http11.py", line 320, in _cb_timeout
    raise TimeoutError("Getting %s took longer than %s seconds." % (url, timeout))
twisted.internet.error.TimeoutError: User timeout caused connection failure: Getting https://wh.lianjia.com/zufang/jiangan/ took longer than 10.0 seconds..
```

redx  
types
id 


https://gaode.com/service/poiTipslite?&city=110000&geoobj=116.298521%7C39.949916%7C116.302786%7C39.951233&words=%E6%88%90%E4%BA%BF%E8%8A%B1%E5%9B%AD%E8%A5%BF%E5%8C%BA
